### PR TITLE
Javascript javascript1 week4

### PR DESCRIPTION
**Pull request by faheem102**

### CODE REVIEW

File: javascript/javascript1/week4/index.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+const personName = []; 
+const toDoList = [];
+let commandArray = [];
```
Comment (aina21) [inline javascript/javascript1/week4/index.js:3]: const

**Review by aina21 — APPROVED** [review]
good job :+1:

**Comment by faheem102** [comment]
thank you aina for your feedback. can i please merge the pull request?

**Comment by aina21** [comment]
yeah :)

**Pull request merged**